src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -2191,6 +2202,27 @@ def GetFabricCheckResult(self) -> int:
         ''' Returns the fabric check result if SetCheckMatchingFabric was used.'''
         return self._fabricCheckNodeId
 
+    '''
+    async def get_commissioning_rcac_data(self):
+        # Await the future until the RCAC data is available
+        try:
+            rcac_data = await asyncio.wait_for(commissioning_future, timeout=60)
+            return rcac_data
+        except asyncio.TimeoutError:
+            raise Exception("Timeout while waiting for RCAC data")
+    '''
+
+    # Function to await the RCAC data
+    async def get_commissioning_rcac_data_async(self):
```

Comment:
why is this async?

Reply:
And, actually - is this used anywhere?

Reply:
Thank you, sorry this was leftover it appears from prior coding down to implement getting RCAC data back from C++ code.